src/shims/unix/linux/eventfd.rs

```diff
@@ -72,6 +96,11 @@ impl FileDescription for Event {
                 Endian::Big => self.counter.to_be_bytes(),
             };
             self.counter = 0;
+            // When any of the event is happened, we check and update the status of all supported flags.
+            // This check is only for linux because other target have no libc epoll flags.
+            if ecx.tcx.sess.target.os == "linux" {
```

Comment:
We already only allow creating `Event` instances on platforms other than linux, so this check should always be `true` I'd think?